esp/scm/ws_decoupledlogging.ecm

```diff
@@ -0,0 +1,57 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+EspInclude(common);
+
+ESPStruct [nil_remove] LogAgentSetting
```

Comment:
If settings for an unsupported agent are requested (i.e., the request agent list is not empty and includes an agent not appropriately configured), it might be nice to have an error status returned in the settings array instead of having the request fail.

Reply:
Good point.